release/build.gradle

```diff
@@ -1,5 +1,7 @@
 plugins {
```

Comment:
separate the build.gradle? one for docker and one for the tar?

Reply:
sure, for now its reusing some of the activities - is it a blocker, if it is i will move them to separate child projects altogether

Reply:
I would split in this pr.